src/runner/driver-test-runner.js

```diff
@@ -289,7 +289,7 @@ export class DriverTestRunner {
 
   _appendBaseUrl(pathname) {
     // protocol ends with a ':' and pathname starts with a '/'
-    const base = `${this.baseUrl.protocol}//${this.baseUrl.hostname}:${this.baseUrl.port}${this.baseUrl.pathname}`;
+    const base = `${this.baseUrl.protocol}://${this.baseUrl.hostname}:${this.baseUrl.port}${this.baseUrl.pathname}`;
```

Comment:
`pathname` should be a URL here but isn't anymore maybe add some typing to ensure it?

https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol

> The protocol property of the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) interface is a string representing the protocol scheme of the URL, including the final `':'`.

The comment should be accurate (this change happens in two places)

Reply:
Found the issue! It looks like the old `agent/cli` file was [coercing some of the "URL" objects into proper JS URL objects](https://github.com/w3c/aria-at-automation-harness/blob/main/src/agent/cli.js#L49-L53). Will replicate this behavior in next commit.